### ISSUE TITLE
locales doesn't work properly on every browser.

### ISSUE THREAD

**Issue by TheSlimReaper925** [nathanreyes/v-calendar#946]
I don't know about any other locales,
but when I choose Georgian ('ka' or 'ka-Ge') Locale, Mozilla and Microsoft Edge does return Georgian values for months and weekdays, but Opera and Chrome doesn't return Georgian values, they return English values. :(

**Comment by maaak7** [comment]
yeah, I have the same problem.. Did you find a solution? @TheSlimReaper925
